.github/workflows/build-and-runtime-test.yml

```diff
@@ -17,16 +17,25 @@ jobs:
       fail-fast: false
       matrix:
         # Paths for each example app we want to test build on
-        path:
-          - angular/angularcli
-          - react/cra
-          - react/cra-ts
-          - react/next
-          - react/vite
-          - react/vite3
-          - vue/vite
-          - vue/vite3
-          - vue/vuecli
+        include:
+          - framework: angular
+            build-tool: angularcli
```

Comment:
I do wish GitHub Actions a less wordy syntax for this, but I think this is the best we can do according to [docs](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) 🥲

Reply:
That's YML for ya

Reply:
That's YML for ya

Reply:
So this fixes the path how? I'm lost here

Reply:
Thanks @wlee221  for the instruction here!